yureks-news/src/components/IndividualArticle.jsx

```diff
@@ -1,18 +1,39 @@
 import { useParams } from "react-router-dom";
-import { getArticleById } from "../functions/functions";
+import { getArticleById, addArticleLike } from "../functions/functions";
 import { useEffect, useState } from "react";
-import Comments from "./Comments";
+// import Comments from "./Comments";
 
 const IndividualArticle = () => {
   const [article, setArticle] = useState({});
-
   const { article_id } = useParams();
 
   useEffect(() => {
     getArticleById(article_id).then((res) => {
       setArticle(res);
     });
-  }, []);
+  }, [article_id]);
+
+  const VotesAdd = () => {
+    const addLike = { inc_votes: 1 };
+    addArticleLike(article_id, addLike)
+      .then((res) => {
+        setArticle(res);
+      })
+      .catch((err) => {
+        return <p>{err}</p>;
+      });
+  };
+
+  const VotesRemove = () => {
+    const RemoveLike = { inc_votes: -1 };
+    addArticleLike(article_id, RemoveLike)
+      .then((res) => {
+        setArticle(res);
+      })
+      .catch((err) => {
+        return <p>{err}</p>;
+      });
+  };
```

Comment:
`VotesAdd` and `VotesRemove` look very similar so I suggest to make one function that will do both things depending on an argument passed in. Make your code dry!